tests/conftest.py

```diff
@@ -38,3 +42,7 @@ def test_case_8_collection() -> Collection:
 def projection_landsat8_item() -> Item:
     path = TestCases.get_path("data-files/projection/example-landsat8.json")
     return Item.from_file(path)
+
+
+def get_data_file(rel_path: str) -> str:
+    return str(here / "data-files" / rel_path)
```

Comment:
This is copied from #1088 I will pull any requested changes on that side into this PR.

Reply:
We should probably migrate any `TestCases.get_path` to `get_data_file` as well, for consistency's sake. Could happen in this or another PR.

Reply:
I think a follow up PR. The intention is to move to fully move to pytest right?

Reply:
Yup (https://github.com/stac-utils/pystac/issues/993, https://github.com/stac-utils/pystac/issues/948), makes sense.